internal/upload/koji/run-koji-container.sh

```diff
@@ -81,7 +81,7 @@ koji_start() {
   ${CONTAINER_RUNTIME} run -d --name org.osbuild.koji.koji --network org.osbuild.koji \
     -v "${SHARE_DIR}:/share:z" \
     -p 80:80 \
-    -p 443:443 \
+    -p 443:4343 \
```

Comment:
I think it is a bit weird to have koji exposed on standard `80` but then the corresponding standard https port (`443`) be composer. Maybe also move koji to `8080` or something?

Reply:
Changed that too.